pkg/utils/additionalProps.go

```diff
@@ -92,7 +91,18 @@ func CheckForAdditionalProps(jsonObj interface{}) (unchanged map[string]interfac
 				unchanged[p] = struct{}{}
 			}
 		default:
-			return
+			addPropTypeName := addProp.Type().Name()
+			if strings.Contains(addPropTypeName, "AdditionalProperty") {
```

Comment:
@frouzbeh - I'm not clear why you had to add this

Reply:
@SeanCondon Because for the SRA the additionalproperty name was AdditionalPropertyStoreId, which didn't hit any other cases and it returned nil. So we weren't able to handle multiple targets in the megapatch. I added this section, so we could retrieve the targetId name (StoreId) , and then get the target (e.g. portland-southeast) from that.

Reply:
got it - thanks